components/cam/src/physics/cam/physpkg.F90

```diff
@@ -430,6 +431,15 @@ subroutine phys_inidat( cam_out, pbuf2d )
     tpert_idx = pbuf_get_index( 'tpert')
     call pbuf_set_field(pbuf2d, tpert_idx, tptr)
 
+    call infld('vmag_gust', fh_ini, dim1name, dim2name, 1, pcols, begchunk, endchunk, &
+         tptr(:,:), found, gridname='physgrid')
+    if(.not. found) then
+       tptr(:,:) = 0._r8
```

Comment:
I wonder how the assignment to tptr after calling infld and not found being used effectively in the sequence of calls in this routine. Anyway, just for consistency, should it be set to 1.0 here?

Reply:
The tptr is not used here. vmag_gust is initialized to 1.0.

Reply:
I wonder if this part of code is necessary, since vmag_gust is always initialized in clubb_tend...